src/test/java/io/logz/guice/jersey/supplier/JerseyServerSupplier.java

```diff
@@ -38,7 +39,7 @@ public static void createServerAndTest(JerseyConfigurationBuilder configurationB
                                            JettyServerCreator jettyServerCreator,
                                            Tester tester) throws Exception {
         int port = FreePortFinder.findFreeLocalPort();
-        configurationBuilder.addPort(port);
+        configurationBuilder.addPortWithHttpConfiguration(port, new HttpConfiguration());
```

Comment:
Let's add a small test that shows I configure jetty with it and it works. Max request header sounds about right